source/code/source/playfab/PlayFabEventBuffer.cpp

```diff
@@ -97,7 +97,7 @@ namespace PlayFab
 
         // create an event packet, set it to the tail->next and move the tail
 
-        const auto currentEventIndex = eventIndex->fetch_add(1, std::memory_order_relaxed);
+        const auto currentEventIndex = static_cast<const uint64_t>(eventIndex->fetch_add(1, std::memory_order_relaxed));
```

Comment:
>const [](start = 51, length = 5)

const is unnecessary here and similar comment about the static_cast being unnecessary as well